vcr-workflow.Rmd

```diff
@@ -0,0 +1,66 @@
+# vcr workflows {#vcr-workflows}
+
+- Re-recording cassettes
+- Making sure vcr is working
+- Make tasks
+
+## Re-recording cassettes {#vcr-re-recording}
+
+There are a number of reasons why you may want to re-record your cassettes:
+
+1. `vcr`/`webmockr` upgrades: you may want to re-record your cassettes if
+you've recently upgraded either of these packages. There should in general
```

Comment:
it's unclear to me whether it's really useful since there should be no problem?

Reply:
what do you mean by "there should be no problem?

Reply:
it's a sentence that you wrote in this paragraph, "There should in general be no problem (...)" so I wonder _when_ it is useful.